.storybook/themes/news.scss

```diff
@@ -633,6 +633,46 @@
 				right-rail-advanced-main-right-rail-bottom: (
 					grid-area: main-right-rail-bottom,
 				),
+				share-bar: (
+					box-shadow: (
+						0 4px 8px 0 rgba(0, 0, 0, 0.2),
+						0 6px 20px 0 rgba(0, 0, 0, 0.19),
+					),
+					border-radius: 0 5px 5px 0,
```

Comment:
sounds like this should use the `--border-radius` token

Reply:
@JackHowa : Now using that token. Thanks!